File: app/src/main/java/com/example/sunkai/heritage/ConnectUsActivity.java
package com.example.sunkai.heritage;

import android.os.Bundle;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;

public class ConnectUsActivity extends AppCompatActivity {

    private ActionBar actionBack;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_connect_us);
        actionBack=getSupportActionBar();
        actionBack.setDisplayHomeAsUpEnabled(true);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()){
            case android.R.id.home:
                finish();
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/com/example/sunkai/heritage/ConnectUsActivity.kt
package com.example.sunkai.heritage

import android.app.AlertDialog
import android.os.Bundle
import android.support.v7.app.ActionBar
import android.support.v7.app.AppCompatActivity
import android.util.Log
import android.view.MenuItem
import android.view.View
import android.widget.TextView
import com.example.sunkai.heritage.Data.GlobalContext
import com.xiaomi.mipush.sdk.MiPushClient

/**
 * Created by sunkai on 2017/12/15.
 */
class ConnectUsActivity : AppCompatActivity() {

    private var actionBack: ActionBar? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_connect_us)
        actionBack = supportActionBar
        actionBack?.setDisplayHomeAsUpEnabled(true)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> finish()
        }
        return super.onOptionsItemSelected(item)
    }
}
